README.md

```diff
@@ -60,13 +60,98 @@ import '@contentful/live-preview/dist/style.css';
 Live Updates from the editor to your applications are currently only supported for [React.js](https://reactjs.org/)
 
 ```tsx
-import { useContentfulLiveUpdates } from "@contentful/live-preview/dist/react"
+import { useContentfulLiveUpdates } from '@contentful/live-preview/dist/react';
 
 // ...
-const updated = useContentfulLiveUpdates(originalData, locale)
+const updated = useContentfulLiveUpdates(originalData, locale);
 // ...
 ```
 
+### Integrating with Gatsby
+
+To use the Contentful Live Preview SDK with Gatsby, you can start with the [gatsby starter contentful homepage](https://www.gatsbyjs.com/starters/gatsbyjs/gatsby-starter-contentful-homepage)
+
+1. Add the @contentful/live-preview package to your Gatsby project by running one of the following commands:
+
+```bash
+yarn add @contentful/live-preview
+```
+
+or
+
+```bash
+npm install @contentful/live-preview
+```
+
+2. In your gatsby-browser.js file, import the live preview styles and initialize the SDK:
+
+```tsx
+import '@contentful/live-preview/dist/style.css';
+import { ContentfulLivePreview } from '@contentful/live-preview';
+
+ContentfulLivePreview.init();
+```
+
+3. In order to tag fields and use live updates, you need to add the contentful_id property to the GraphQL schema. For example, to extend the HomepageHero interface:
+
+```graphql
+interface HomepageHero {
```

Comment:
it this the part in the gatsby-node.js?
Then I had to also adjust this one: `ContentfulHomepageHero` :(

Reply:
Ah yes I will add it as well 👌